Code/GraphMol/GeneralizedSubstruct/XQMol.cpp

```diff
@@ -19,6 +19,8 @@
 
 #include "XQMol.h"
 
+#include "GraphMol/Fingerprints/Fingerprints.h"
```

Comment:
I think you also need to add a dependency on the Fingerprints library to GeneralizedSubstruct

Reply:
I added the Fingerprints library to CMakeLists.txt.